<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GtPreferencesCommandlineTemplates</class>
 <widget class="QWidget" name="GtPreferencesCommandlineTemplates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="machine"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="shell"/>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelOs">
     <property name="text">
      <string>OS Type:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <widget class="QGroupBox" name="testBox">
     <property name="title">
      <string>Test: (save, close and re-open this dialog before)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Template:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QLineEdit" name="testToolversion"/>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="buttonCheckTemplateExists">
        <property name="text">
         <string>Check template exists</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Version:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="testToolname"/>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QLineEdit" name="testTemplatename"/>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="buttonTestTemplate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Tool Name:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2" colspan="2">
       <widget class="QLabel" name="labelTemplateFound">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelShell">
     <property name="text">
      <string>Default Shell:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" rowspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="dir">
     <property name="toolTip">
      <string extracomment="The path to the directory where the templates are stored"/>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="buttonOpenPathInExplorer">
     <property name="text">
      <string>Open in Explorer</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="os"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelPath">
     <property name="text">
      <string>Templates Path:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelMachine">
     <property name="text">
      <string>Machine:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="buttonRestoreDefaults">
     <property name="text">
      <string>Restore Defaults</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>os</tabstop>
  <tabstop>dir</tabstop>
  <tabstop>machine</tabstop>
  <tabstop>shell</tabstop>
  <tabstop>testToolname</tabstop>
  <tabstop>testToolversion</tabstop>
  <tabstop>testTemplatename</tabstop>
  <tabstop>buttonCheckTemplateExists</tabstop>
  <tabstop>buttonTestTemplate</tabstop>
  <tabstop>buttonOpenPathInExplorer</tabstop>
  <tabstop>buttonRestoreDefaults</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
